analytics/src/docker/Dockerfile

```diff
@@ -1,11 +1,17 @@
 FROM jetty:9-jre11
+USER root
+RUN mkdir -p /docker-entrypoint.d
+RUN chown jetty /docker-entrypoint.d
```

Comment:
Could you change it to `chown jetty:jetty /docker-entrypoint.d` so that it is more consistent with the other `COPY --chown=jetty:jetty` and other chown in other applications: `chown jetty:jetty`: https://github.com/georchestra/geonetwork/blob/startup-script/web/src/docker/Dockerfile#L15C5-L15C22